### ISSUE TITLE
PDF/UA Labels with line breaks: words of two lines are merged in screen reader

### ISSUE THREAD

**Issue by hvbtup** [eclipse-birt/birt#2446]
### Description
Data items support line breaks generally (labels are technically similar, but you cannot enter line breaks).
The rendering is OK, but when creating PDF/UA, the line breaks get lost.
This results in two adjacent "words" merged to one word, e.g. in a German address like

ACME Chemicals
Toon Street 12
34567 Toontown

... the screen reader "sees" ACME ChemicalsToon Street 1234567 Toontown.

This is confusing for the numbers in particular.

### Environment
BIRT master branch with some other uncommited changes in place,
but this should be reproducible in master as well.

[issue-2446.zip](https://github.com/user-attachments/files/30590362/issue-2446.zip)

**Comment by hvbtup** [comment]
I'll provide a PR with a fix.

**Comment by hvbtup** [comment]
I modified the example to generate a label with the lines "12" and "345" and noticed that the SPAN tag used is ignored by the screen readers, so they read it as 12 thousand 345 instead of two words 12 ans 345. So the change from the PR 2453 shows where we can detect the line break, but it doesn't have the intended effect.

ATM I am a bit unsure what a correct and simple solution would be.

One idea is to generate keep the space character in the line-breaking process, but to not use it in the width calculation.
While this idea seems logical, I spent much time to optimize the line-breaking process back then, and that process is quite difficult.

That's why I'd prefer a solution which requires only changes to TextLineArea.java.
